frontend/src/components/Footer/Footer.tsx

```diff
@@ -4,143 +4,80 @@ import { ActionContext } from "../../hooks";
 
 function Footer() {
   const { setModalConfig } = useContext(ActionContext);
-  const signupFormUrl = `${process.env.REACT_APP_SIGNUP_FORM_URL}`;
+  /*const signupFormUrl = `${process.env.REACT_APP_SIGNUP_FORM_URL}`;
```

Comment:
These were removed from the copy by madeline. We should remove them here as well since they are not used.